UI/Selenium/Pages/BookingListPage.cs

```diff
@@ -14,5 +14,9 @@ public class BookingListPage
         public static By HearingDetailsRowSpecific(string text) => By.XPath($"//div[text()[contains(.,'{text}')]]");
         public static By SearchCaseTextBox = By.Id("caseNumber");
         public static By SearchButton => By.Id("searchButton");
+        public static By ConfirmedButton => By.XPath("//*[contains(text(),'Confirmed')]");
```

Comment:
id is preferred

Reply:
Tried With id was failing in few occasions and Xpath is working fine.